python/cudf/cudf/core/column/categorical.py

```diff
@@ -309,22 +307,20 @@ def add_categories(self, new_categories, **kwargs):
             [old_categories.dtype, new_categories.dtype], []
         )
 
-        new_categories = new_categories.astype(common_dtype, copy=False)
-        old_categories = old_categories.astype(common_dtype, copy=False)
+        new_categories = new_categories.astype(common_dtype)
+        old_categories = old_categories.astype(common_dtype)
```

Comment:
Because that parameter ends up never getting used. These lines call [`ColumnBase.astype()`](https://github.com/rapidsai/cudf/blob/branch-0.17/python/cudf/cudf/core/column/column.py#L994-L1014), which in turn calls [`as_categorical_column`](https://github.com/rapidsai/cudf/blob/branch-0.17/python/cudf/cudf/core/column/column.py#L1016-L1056). I didn't find the use of `copy` in either of those

Reply:
Just a note: `Series.astype()` will continue to need the `copy` parameter: https://pandas.pydata.org/docs/reference/api/pandas.Series.astype.html